autotest/gcore/rat.py

```diff
@@ -135,7 +135,43 @@ def test_rat_3():
     ds = None
 
     gdal.GetDriverByName('GTiff').Delete('/vsimem/rat_3.tif')
-##############################################################################
 
+###############################################################################
+# Edit an existing RAT (#3783)
+
+
+def test_rat_4():
+
+    ds = gdal.GetDriverByName('GTiff').Create('/tmp/rat_4.tif', 1, 1)
```

Comment:
ah /tmp will not work on windows. Could you use a tmpfilename = '/vsimem/rat_4.tif' variable instead ?